index.js

```diff
@@ -45,6 +48,11 @@ if (process.env.GOOGLE_MAPS_API_KEY && process.env.GEONAME_CLIENT_ID) {
   console.warn("Geo keys missing. Not starting geo routes.");
 }
 
+app.post(
```

Comment:
New endpoint name for clarity and to allow for future neighborhood-finder endpoints